.tav.yml

```diff
@@ -211,13 +211,7 @@ cassandra-driver:
     - node test/instrumentation/modules/cassandra/cassandra.js
 
 restify:
-  # All versions before 4.4 are unsupported. Additionally,
-  # there are several broken releases that need to be skipped:
-  # - 6.2.0
-  # - 6.0.1
-  # - 6.0.0
-  # - 5.0.0
-  versions: '>=4.4 <5 || >5 <6 || >6.0.1 <6.2 || >6.2'
```

Comment:
FYI: During research on this bug I also found that this version string does't produce the expected range:

```js
semver.validRange('>=4.4 <5 || >5 <6 || >6.0.1 <6.2 || >6.2')
// '>=4.4.0 <5.0.0||>=6.0.0 <6.0.0||>6.0.1 <6.2.0||>=6.3.0'
```

As you can see, `>5` is not translated into `>5.0.0` but into `>=6.0.0`. So our TAV tests have been ignoring a big part of the version range when testing.

Reply:
Hmm this might actually explain why I can't get the TAV tests to run now. They refuse to install v5.2.0 due to a weird bug. This might have been a problem all along and we just never knew because we didn't run the tests with 5.x

Reply:
Weird. I could have sworn it was testing 5.x properly. 🤔

Reply:
I found the bug: restify@6.2.0 had a (sub)dependency to npm@2 which then took over when tav tried to continue to test v6.0.1. That's why 6.2.0, 6.0.1, and 6.0.0 failed. I updated test-all-versions to not fail even if one of the modules it installs depends on npm. So now all Restify tests pass without us having to skip any tests.